.travis.yml

```diff
@@ -44,6 +63,5 @@ before_script:
 
 script:
   - make ci
-  - black . --check
 after_success:
-  - codecov
+  - if [ -z ${CODECOV} ]; then echo "Skipping codecov submit"; else codecov; fi
```

Comment:
Diese Zeile stellt sicher, dass `codecov` nur ausgeführt wird, wenn die `CODECOV` Variable gesetzt ist. Dadurch wird `codecov` nicht in der Linter Stage ausgeführt.